src/sider/sykmeldinger/sykmelding/avvisteSykmeldinger/AvvistSykmeldingAlert.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from "react";
+import { SykmeldingOldFormat } from "@/data/sykmelding/types/SykmeldingOldFormat";
+import * as avvisningsregelnavn from "../../../../utils/sykmeldinger/avvisningsregelnavn";
+import { Alert, BodyShort, Box, Heading, List } from "@navikt/ds-react";
+
+const avvistSykmeldingFilter = (sykmelding: SykmeldingOldFormat) => {
+  return sykmelding.behandlingsutfall.ruleHits.filter((rule) => {
+    return rule.ruleStatus === null || rule.ruleStatus === "INVALID";
+  });
+};
+
+interface BegrunnelseTekstProps {
```

Comment:
Ikke noe vi _må_ gjøre, men jeg er fan av å ta med raske små endringer som bedre følger konvensjoner vi har blitt enige(?) om å bruke 😊🤌
```suggestion
interface Props {
```

Reply:
Hm, her er det jo flere komponenter i samme fil også, vet ikke hva vi ønsker skal være "hovedkomponenten"? Det blir kanskje fort `AvvistSykmeldingAlert`